go/signedexchange/signedexchange_test.go

```diff
@@ -227,13 +227,15 @@ func TestSignedExchangeBannedCertUrlScheme(t *testing.T) {
 	}
 }
 
-func TestVerify(t *testing.T) {
+func createTestExchange(t *testing.T) (e *Exchange, s *Signer, certBytes []byte) {
 	u, _ := url.Parse("https://example.com/")
-	header := http.Header{}
-	header.Add("Content-Type", "text/html; charset=utf-8")
+	reqHeaders := http.Header{}
+	reqHeaders.Add(":method", "GET")
```

Comment:
The current situation around `Exchange.RequestHeaders` is quite tricky. We might want to resolve it first.
`func (*Exchange) encodeRequest()` assumes `":method"` key to not be in the `RequestHeaders` map, while `func (*Exchange) decodeRequest()` will populate the key.

Reply:
OK, created #344.